README.md

```diff
@@ -7,6 +7,19 @@ To get started, you will need an installation of
 [Docker Compose](https://docs.docker.com/compose/install/) on the host on
 which you will run your Puppet Infrastructure.
 
+## Required versions
+
+* Docker Compose - must support `version: '3'` of the compose file format, which requires Docker Engine `1.13.0+`. [Full compatibility matrix](https://docs.docker.com/compose/compose-file/compose-versioning/)
+  * Linux is tested with docker-compose `1.22`
+  * Windows is tested with `docker-compose version 1.24.0dev, build 4bd93b95`
+* Docker Engine support is only tested on versions newer than `17.09.0-ce`
+  * Linux is tested with (client and server) `17.09.0-ce` using API version `1.32` (`Git commit:   afdb6d4`)
+  * Windows is tested with newer nightly versions that enable LCOW support / fix bugs in the Docker runtime
+    * Client `master-dockerproject-2019-01-08` using API version `1.40` (`Git commit:        d04b6165`)
+    * Server `master-dockerproject-2019-01-08` using API version `1.40 (minimum version 1.24)` (`Git commit:        77df18c`) with `Experimental: true`
+
```

Comment:
do we want to include osx too? this is what I've got

```
morgan@pond % docker version
Client: Docker Engine - Community
 Version:           18.09.1
 API version:       1.39
 Go version:        go1.10.6
 Git commit:        4c52b90
 Built:             Wed Jan  9 19:33:12 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.1
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       4c52b90
  Built:            Wed Jan  9 19:41:49 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

```
morgan@pond % docker-compose version
docker-compose version 1.23.2, build 1110ad01
docker-py version: 3.6.0
CPython version: 3.6.6
OpenSSL version: OpenSSL 1.1.0h  27 Mar 2018
```

Reply:
Same as @underscorgan - thanks!